### PR TITLE
Bug1368610 - stylo: implement primitive (bit, uXX) type of discrete animatable properties

### PR DESCRIPTION
**Pull request by dadaa**
This PR is to fix https://bugzilla.mozilla.org/show_bug.cgi?id=1368610

---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors

- [X] There are tests for these changes. The tests will land in dom/animation/test of m-c. Test code is patch 6 of https://bugzilla.mozilla.org/show_bug.cgi?id=1368610

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/17253)

### CODE REVIEW

**Comment by dadaa** [comment]
These patches have been reviewed by @hiikezoe

File: components/style/properties/longhand/ui.mako.rs [hunk 1]
```diff
@@ -83,4 +83,21 @@ ${helpers.single_keyword("-moz-window-shadow", "none default menu tooltip sheet"
             _ => Err(()),
         }
     }
+
+    #[cfg(feature = "gecko")]
```
Comment (nox) [inline components/style/properties/longhand/ui.mako.rs]: Is that really needed? This is a gecko-only property.

Comment (dadaa) [inline components/style/properties/longhand/ui.mako.rs]: @nox thank you for your comment!
Ah, it looks we can drop.
I will re-push after fix this.

**Comment by emilio** [comment]
Also, you should probably configure your git email. The commits are done with `daisuke@daisukenoMacBook-Pro.local`.

**Comment by dadaa** [comment]
@emilio thank you.
So, should I pull request again?

**Comment by emilio** [comment]
@bors-servo r=hiikezoe,nox,emilio

Looks fine, thanks a lot! :)

**Pull request merged**